library/odoo

```diff
@@ -1,9 +1,9 @@
 Maintainers: Christophe Monniez <moc@odoo.com> (@d-fence)
 GitRepo: https://github.com/odoo/docker
-GitCommit: d04d885eea6b8f60ada727fcc498f5a8fd44da51
+GitCommit: 9b5e2cf27bcc23da4096c40b7369d0d43a4d90c6
```

Comment:
The Dockerfile diff looks the same. 😕 

```diff
+    "ppc64le") WKHTMLTOPDF_ARCH=ppc64el && WKHTMLTOPDF_SHA=7ed8f6dcedf5345a3dd4eeb58dc89704d862f9cd  ;; \
```

Ah, I think the `GitCommit` needs to be updated to https://github.com/odoo/docker/commit/3864188c94c33dace7d4b5f6767e00a4328fe08c?

Reply:
🤦🏻 omg, you are right.
Changed now.